lib/native/binding.js

```diff
@@ -74,5 +74,15 @@ if (binding == undefined) {
     throw new Error("AWS CRT binary not present in any of the following locations:\n\t" + search_paths.join('\n\t'));
 }
 
+import crt_native from "./binding"
+/** As Electron would shutdown the node process on exit, and eventually causing "segfault" on threadsafe function operation.
```

Comment:
Trivial: "Electron will shutdown the node process on exit, which causes the threadsafe function to segfault. To prevent the segfault we disable the threadsafe function on node process exit."